lib/node_modules/@stdlib/random/docs/types/index.d.ts

```diff
@@ -173,6 +874,40 @@ interface Namespace {
 	* // returns <ndarray>
 	*/
 	uniform: typeof uniform;
+
+	/**
+	* Generates pseudorandom numbers drawn from a Weibull distribution.
+	*
+	* @param shape - output shape
+	* @param k - scale parameter
+	* @param lambda - shape parameter
```

Comment:
Parameter names were mixed up for `random/weibull`.

```suggestion
	* @param k - shape parameter
	* @param lambda - scale parameter
```